.demo/env/charts/gatekeeper.yaml

```diff
@@ -0,0 +1,75 @@
+charts:
+    gatekeeper:
+        enabled: false
+        constraints:
+            - parameters:
```

Comment:
please use policyName as key

Reply:
Ah, yes.. well initially it did not matter much, having to decide which data structure to use for a list of objects made sense with a List - but I did find it hard in some cases to work with a list structure, especially when dynamically loading values with the tools for shell scripts.
I am happy to change it to a map, it does make more sense because the policy blocks need to be unique as well.